### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by BryanHerrera19** [BryanHerrera19/COMP_055-project-tanks#1]
This new feature will add a leaderboard to the game. Each time a player reaches the last level or loses, they will be prompted to enter their username where their score and name will be added to a file and saved into the game directory. When they go to the leaderboard on the main screen they will then be able to see the leaderboard with each name and high score in order from greatest to least.
 A new class will be needed for the actual scoreboard menu to show up in the leaderboard page as well as a new UI for entering a name at the end of a game. There may also be a need for a couple of functions in order to parse the data as well as sort the scores from greatest to least. There will also need to be a method to read and write files.

**Comment by onzfonz** [comment]
Yep!  Do this one!  The important part will be the files part. Good luck!
